ui/components/app/confirm-hexdata/README.mdx

```diff
@@ -9,7 +9,3 @@ Confirm Hex Data is used on confirmation screen to display transaction data.
   <Story id="components-app-ConfirmHexData--default-story" />
   <Story id="components-app-ConfirmHexData--data-hex-component-story" />
 </Canvas>
-
-## Props
-
-<ArgsTable of={ConfirmHexData} />
```

Comment:
I think we can just remove this file altogether and remove the reference from the storybook file